apps/showcase/src/app/placeholder/placeholder.component.ts

```diff
@@ -11,6 +11,21 @@ import { IN_PAGE_NAV_PRES_DIRECTIVES, type InPageNavLink, InPageNavLinkDirective
 import { environment } from '../../environments/environment.development';
 import { TripFactsService } from '../../facts';
 
+const prepareRulesEngine = async () => {
+  inject(TripFactsService).register();
+  inject(RulesEngineRunnerService).actionHandlers.add(inject(PlaceholderRulesEngineActionHandler));
+  const path = await firstValueFrom(
+    inject(DynamicContentService).getContentPathStream(
+      `${!environment.production ? 'assets/' : ''}rules/rulesets.json`
+    )
+  );
+
+  const resultCall = await fetch(path);
+  const result = await resultCall.json();
+
+  inject(Store<RulesetsStore>).dispatch(setRulesetsEntities({ entities: result.rulesets }));
```

Comment:
is this really working ? I would assume that after the first `await` we are outside the injection context of Angular